p2p/upnp/upnp.go

```diff
@@ -285,7 +285,7 @@ func soapRequest(url, function, message, domain string) (r *http.Response, err e
 	    defer r.Body.Close()
 	}*/
 
-	if r.StatusCode >= 400 {
+	if r.StatusCode != http.StatusOK {
```

Comment:
I think that the level of abstraction will be higher if you call soapRequest and handle it.
However, if there are no plans to add soapRequests other than "status:200" in the future, this implementation looks good.(for example 204 etc...)

Reply:
Currently, I think there is nothing. In the future, It's enough that we should reconsider it again.

Reply:
Thank you for your opinion!